xikolo-ios/Views/NewsArticleCell.swift

```diff
@@ -13,11 +13,28 @@ class NewsArticleCell : UITableViewCell {
     @IBOutlet weak var titleView: UILabel!
     @IBOutlet weak var descriptionView: UILabel!
     @IBOutlet weak var readStateView: UIView!
+    @IBOutlet weak var dateView: UILabel!
+    @IBOutlet weak var courseView: UILabel!
 
     func configure(newsArticle: NewsArticle) {
         readStateView.backgroundColor = Brand.TintColor
+        
+        let dateFormatter = NSDateFormatter()
```

Comment:
I meant put these lines into the `if let you already have`. It's not necessary to execute this code if `published_at` is `nil`.